docs/source/start.rst

```diff
@@ -168,13 +168,14 @@ libffi. On a Debian-derivative system, this could be accomplished with
 
 ::
 
-   sudo apt-get install liblapack-dev libblas-dev libffi-dev
+   sudo apt-get install liblapack-dev libblas-dev libffi-dev libzmq3-dev
 
 Or on any rhel-derivative systems (e.g. Amazon Linux) with
 
 ::
 
-   sudo yum install -y lapack-devel blas-devel
+   sudo yum install -y lapack-devel blas-devel epel-release
+   sudo yum install -y zeromq3-devel
```

Comment:
do these have to be two different commands?

Reply:
oh probably because of epel

Reply:
Yep, exactly because of that. In favor of a solution that doesn't need two lines, I just don't know what it is.